app/src/main/java/org/cru/godtools/ui/tools/ToolsContainerFragment.kt

```diff
@@ -0,0 +1,19 @@
+package org.cru.godtools.ui.tools
+
+import android.os.Bundle
+import androidx.fragment.app.viewModels
+import dagger.hilt.android.AndroidEntryPoint
+import org.cru.godtools.R
+import org.cru.godtools.databinding.ToolsContainerFragmentBinding
+import org.cru.godtools.fragment.BasePlatformFragment
+
+@AndroidEntryPoint
+class ToolsContainerFragment : BasePlatformFragment<ToolsContainerFragmentBinding>(R.layout.tools_container_fragment) {
+
+    private val dataModel: ToolsContainerFragmentDataModel by viewModels()
+    override fun onBindingCreated(binding: ToolsContainerFragmentBinding, savedInstanceState: Bundle?) {
+        super.onBindingCreated(binding, savedInstanceState)
+        binding.refresh.setupSwipeRefresh()
+        binding.hasSpotLight = dataModel.hasSpotlight
```

Comment:
try and keep case on names consistent. because `spotlight` is a single word, you shouldn't have a capital `L` in the middle of the name.